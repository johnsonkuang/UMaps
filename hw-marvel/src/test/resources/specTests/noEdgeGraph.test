# tests a graph with no edges

LoadGraph graph noEdgeGraph.tsv

ListNodes graph

ListChildren graph 1
ListChildren graph 2
ListChildren graph 3

FindPath graph 1 2
FindPath graph 2 3
FindPath graph 3 1