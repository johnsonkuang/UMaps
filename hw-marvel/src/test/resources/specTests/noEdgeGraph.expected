# tests a graph with no edges

loaded graph graph

graph contains: 1 2 3

the children of 1 in graph are:
the children of 2 in graph are:
the children of 3 in graph are:

path from 1 to 2:
no path found
path from 2 to 3:
no path found
path from 3 to 1:
no path found